android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/components/MultipleChoicesAnswer.kt

```diff
@@ -0,0 +1,102 @@
+package co.nimblehq.ic.kmm.suv.android.ui.components
+
+import android.annotation.SuppressLint
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.Button
+import androidx.compose.material.ButtonDefaults
+import androidx.compose.material.Text
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import co.nimblehq.ic.kmm.suv.android.R
+import co.nimblehq.ic.kmm.suv.android.ui.theme.Typography
+
+@SuppressLint("MutableCollectionMutableState")
+@Composable
+fun MultipleChoicesAnswer(
+    choices: List<String>,
+    onIndexesChange: (Set<Int>) -> Unit,
+    currentIndexes: Set<Int> = mutableSetOf()
+) {
+    var selectedIndexes by remember { mutableStateOf(currentIndexes) }
+    Column(modifier = Modifier.padding(horizontal = 80.dp)) {
+        choices.forEachIndexed { index, choice ->
+            val isHighlight = selectedIndexes.contains(index)
+            Button(
+                onClick = {
+                    if (selectedIndexes.contains(index)) {
+                        selectedIndexes = selectedIndexes.minus(index)
+                    } else {
+                        selectedIndexes = selectedIndexes.plus(index)
+                    }
```

Comment:
optional

```suggestion
                    selectedIndexes = if (selectedIndexes.contains(index)) {
                        selectedIndexes.minus(index)
                    } else {
                        selectedIndexes.plus(index)
                    }
```

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/118/commits/9596bc9bf072bfdede796aea64e6e6536a022a88